realm/realm-library/src/androidTest/java/io/realm/rule/RunInLooperThread.java

```diff
@@ -76,6 +76,11 @@ protected void before() throws Throwable {
     @Override
     protected void after() {
         super.after();
+
+        // Waits all async tasks done to ensure successful deleteRealm call.
```

Comment:
Wait for all async tasks to have completed to ensure a successful deleteRealm call.
This will throw when if it times out. <I'm not sure I understand that last sentance?>